backend/test/routes/amendments.test.js

```diff
@@ -0,0 +1,47 @@
+const model = require("../../src/models/amendments.js");
+const { testRoutes, routeTypes } = require("./index.js");
+
+jest.mock("../../src/models/amendments.js");
+
+const capability = ["amendments_read_all"];
```

Comment:
i'm about to break all your tests. :( I guess the last one to breaks it fixes!!

Reply:
Haha yeah I figured the capabilities ticket would break all of this but it should be a fairly easy fix.